source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -1212,69 +820,48 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
             }
         }
 
-        // Create an AutomationPropertiesStatics object so we can set the accessibility properties that label allow us to use.
-        ComPtr<IAutomationPropertiesStatics> automationPropertiesStatics;
-        RETURN_IF_FAILED(
-            GetActivationFactory(HStringReference(RuntimeClass_Windows_UI_Xaml_Automation_AutomationProperties).Get(),
-                                 &automationPropertiesStatics));
-
+        // TODO: do I need this
```

Comment:
@paulcam206 could you please have a look at this?

Reply:
same as above -- I don't think so :)